packages/plugin-server-session/tracker.js

```diff
@@ -30,13 +30,10 @@ module.exports = class SessionTracker extends Emitter {
   }
 
   _summarize () {
-    const thisMin = dateToMsKey(new Date())
     const summary = []
     this._sessions.forEach((val, key) => {
-      if (key !== thisMin) {
```

Comment:
Is this check not needed anymore?

Reply:
ah I meant to explain this…

So the tests were failing to send any session summaries even though I brought the interval right down to a second. This turned out to be the thing that was preventing it – it refused to the summary for the current minute, on the premise that more sessions could be added before the minute is over. I checked with @bullmo and it turns out that it doesn't matter if we send multiple summaries for the same minute – they will be summed together in the sessions service. So tl;dr, no this check is not needed, but thanks for making sure I explained it 🦅 👀